### ISSUE TITLE
Any insights of using PMI?

### ISSUE THREAD

**Issue by 1049451037** [yao8839836/text_gcn#21]
Hi, I am very interested in your work. I noticed that you said that "PMI achieves better results than using word co-occurrence count in our preliminary experiments". Is there any experiment result to share? And are there any other attempts besides these two choices experiments you did?

**Comment by yao8839836** [comment]
@1049451037 

Hi thanks for your interests.

Followings are test accuracy scores with word co-occurrence count, I didn't run the model 10 times with this setting, so I only obtained one accuracy score here.

window size 20:

R8: 0.94335
MR: 0.73804

window size 5:

R8: 0.94016
MR: 0.74648

Recently, we also found Text GCN performs relatively well without word-word edges. Followings are 
test accuracy scores with doc-word TFIDF edges only:

20ng | ohsumed | MR | R8 | R52

0.8428 | 0.66411 | 0.76112 | 0.95614 | 0.92406
